lxd/instance_sftp.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"io"
+	"net"
+	"net/http"
+	"sync"
+
+	"github.com/gorilla/mux"
+	log "gopkg.in/inconshreveable/log15.v2"
+
+	"github.com/lxc/lxd/lxd/cluster"
+	"github.com/lxc/lxd/lxd/instance"
+	"github.com/lxc/lxd/lxd/response"
+	"github.com/lxc/lxd/shared/api"
+	"github.com/lxc/lxd/shared/logger"
+	"github.com/lxc/lxd/shared/logging"
+	"github.com/lxc/lxd/shared/tcp"
+)
+
+func instanceSFTPHandler(d *Daemon, r *http.Request) response.Response {
```

Comment:
@tomponline you're missing the Swagger comment here (and associated swagger update commit)